### ISSUE TITLE
Config for parler TTS

### ISSUE THREAD

**Issue by Vasu-yadav** [SherryS997/parler-tts-dataset#1]
Hey @SherryS997 I have been working on training parler TTS for HINDI using the given training recipes on parler-TTS github. I have tried changing Hyperparamters but the loss doesn't seem to reduce.
<img width="542" alt="Screenshot 2024-10-17 at 8 36 45 AM" src="https://github.com/user-attachments/assets/561e2cb7-f663-48bb-ba44-57bd19414adc">

Can you please guide me on what changes to make in original recipe and what steps to follow.

**Comment by SherryS997** [comment]
![image](https://github.com/user-attachments/assets/5e38ed02-9547-439c-b372-f1da9789ddd8)

The loss stagnates a lot at around 3.9 early on, but it doesn't mean that the model is not learning. At least this is what I've been seeing in our experiments. Let it train. Do some inference later and see how the model performs. All the best.

**Comment by SherryS997** [comment]
As for the changes we made, we changed the tokenizer for the prompt. The vanilla parler tts uses FlanT5 tokenizer which the language we wanted to support couldn't. So we trained our own tokenizer. In your case, you can just use the llama 3 tokenizer. Should be good enough.
